upup/models/cloudup/resources/addons/certmanager.io/k8s-1.16.yaml.template

```diff
@@ -5187,20 +5202,20 @@ apiVersion: v1
 kind: Service
 metadata:
   name: cert-manager-webhook
-  namespace: "kube-system"
+  namespace: kube-system
   labels:
     app: webhook
     app.kubernetes.io/name: webhook
     app.kubernetes.io/instance: cert-manager
     app.kubernetes.io/component: "webhook"
-    app.kubernetes.io/version: "v1.9.1"
+    app.kubernetes.io/version: "v1.10.1"
 spec:
   type: ClusterIP
   ports:
   - name: https
     port: 443
     protocol: TCP
-    targetPort: 10250
+    targetPort: "https"
```

Comment:
Not quite sure why this was left to `10250` since this change has been around for some time.